src/theme/components/structuredContent.ts

```diff
@@ -1,34 +1,47 @@
 const StructuredContent = {
-  parts: ["image", "imageContainer"],
-  baseStyle: ({ imagePosition, noImageWrapper }) => {
+  parts: ["image", "imageFigure", "imageWrapper"],
+  baseStyle: ({ imageAspectRatio, imagePosition }) => {
     const styles: { maxWidth?: string } = {};
+    const wrapperStyles = {
+      float: imagePosition !== "center" ? imagePosition : undefined,
+      marginBottom: "m",
+      marginLeft:
+        imagePosition === "center"
+          ? null
+          : imagePosition === "left"
+          ? "0"
+          : "m",
+      marginRight:
+        imagePosition === "center"
+          ? null
+          : imagePosition === "left"
+          ? "m"
+          : "0",
+      width: imagePosition !== "center" ? "100%" : undefined,
+      ...styles,
+    };
     if (imagePosition === "center") {
       styles.maxWidth = "100%";
     }
     return {
-      imageContainer: {
-        float: imagePosition !== "center" ? imagePosition : undefined,
+      imageFigure: wrapperStyles,
+      imageWrapper: {
+        ...wrapperStyles,
         marginBottom: "m",
-        marginLeft:
-          imagePosition === "center"
-            ? null
-            : imagePosition === "left"
-            ? "0"
-            : "m",
-        marginRight:
-          imagePosition === "center"
-            ? null
-            : imagePosition === "left"
-            ? "m"
-            : "0",
-        width: imagePosition !== "center" ? "100%" : undefined,
-        ...styles,
       },
       image: {
         // To handle the case when the image does not have a container,
         // e.g. when there is no image caption or credit.
         float: imagePosition !== "center" ? imagePosition : undefined,
-        marginBottom: noImageWrapper ? "m" : null,
+        marginLeft:
+          imagePosition === "right" && imageAspectRatio === "original"
+            ? "m"
+            : undefined,
+        marginRight:
+          imagePosition === "left" && imageAspectRatio === "original"
+            ? "m"
+            : undefined,
+        marginBottom: "var(--nypl-space-m) !important",
```

Comment:
This style needs some logic applied.  The way this is written now, the bottom margin will always be applied to the image element and there is a gap between the image and the image caption.  

However, I see above the style was originally written as:
```marginBottom: noImageWrapper ? "m" : null,```

That seems like it would have been correct.  Why the change?